spec/controllers/openid_connect/authorization_controller_spec.rb

```diff
@@ -1076,7 +1076,7 @@
         sp_request_id = ServiceProviderRequestProxy.last.uuid
 
         expect(session[:sp]).to eq(
-          aal_level_requested: nil,
+          aal_level_requested: 1, # ToDo: Get Hooper's opinion
```

Comment:
This is a good catch. I think this could be problematic here: https://github.com/18F/identity-idp/blob/889e7a6e351ac93e35493bf6fb0b255f88336176/app/policies/service_provider_mfa_policy.rb#L68

That said, this is there to handle the `aal3` vs `phishing_resistant` ACR values and to add service provider defaults. The `AuthnContextResolver` handles both of those so I think this should be fine. We may want to consider a second task to take a look at the `ServiceProviderMfaPolicy` and trim dead code.

Reply:
Gonna leave it as is, then. Will generate that second task.